psc/src/main/java/com/pinterest/psc/producer/transaction/KafkaTransactionManagerOperator.java

```diff
@@ -0,0 +1,271 @@
+package com.pinterest.psc.producer.transaction;
```

Comment:
I'd suggest following the same pattern as before for the packaging. I.e. for this one it should go as `...src/main/java/com/pinterest/psc/producer/transaction/kafka/TransactionManagerOperator.java`